relay-clients/client-asset-hub-westend/src/lib.rs

```diff
@@ -0,0 +1,127 @@
+	RefundBridgedParachainMessagesSchema,
+};
+
+pub type CheckMetadataHash = GenericTransactionExtensionSchema<Mode, Option<[u8; 32]>>;
+
+pub type TransactionExtension = SuffixedCommonTransactionExtension<(
+	CheckMetadataHash,
+	BridgeRejectObsoleteHeadersAndMessages,
+	RefundBridgedParachainMessagesSchema,
+)>;
+
+pub type RuntimeCall = runtime_types::asset_hub_westend_runtime::RuntimeCall;
+pub type BridgeMessagesCall = runtime_types::pallet_bridge_messages::pallet::Call;
+type UncheckedExtrinsic =
+	bp_asset_hub_westend::UncheckedExtrinsic<RuntimeCall, TransactionExtension>;
+
+/// AssetHubWestend chain definition
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct AssetHubWestend;
+
+impl UnderlyingChainProvider for AssetHubWestend {
+	type Chain = bp_asset_hub_westend::AssetHubWestend;
+}
+
+impl Chain for AssetHubWestend {
+	const NAME: &'static str = "AssetHubWestend";
+	const BEST_FINALIZED_HEADER_ID_METHOD: &'static str =
+		bp_asset_hub_westend::BEST_FINALIZED_ASSET_HUB_WESTEND_HEADER_METHOD;
+	const FREE_HEADERS_INTERVAL_METHOD: &'static str =
+		bp_asset_hub_westend::FREE_HEADERS_INTERVAL_FOR_ASSET_HUB_WESTEND_METHOD;
+	const AVERAGE_BLOCK_INTERVAL: Duration = AVERAGE_BLOCK_INTERVAL;
+
+	type SignedBlock = bp_asset_hub_westend::SignedBlock;
+	type Call = RuntimeCall;
+}
+
+impl ChainWithBalances for AssetHubWestend {
+	fn account_info_storage_key(account_id: &Self::AccountId) -> StorageKey {
+		bp_asset_hub_westend::AccountInfoStorageMapKeyProvider::final_key(account_id)
+	}
+}
+
+impl ChainWithTransactions for AssetHubWestend {
+	type AccountKeyPair = sp_core::sr25519::Pair;
+	type SignedTransaction = UncheckedExtrinsic;
+
+	fn sign_transaction(
+		param: SignParam<Self>,
+		unsigned: UnsignedTransaction<Self>,
+	) -> Result<Self::SignedTransaction, SubstrateError> {
+		let raw_payload = SignedPayload::new(
+			FakeDispatchable::from(unsigned.call),
+			TransactionExtension::from_params(
+				param.spec_version,
+				param.transaction_version,
+				unsigned.era,
+				param.genesis_hash,
+				unsigned.nonce,
+				unsigned.tip,
+				((Mode::Disabled, (), ()), (None, (), ())),
+			),
+		)?;
+
+		let signature = raw_payload.using_encoded(|payload| param.signer.sign(payload));
+		let signer: sp_runtime::MultiSigner = param.signer.public().into();
+		let (call, extra, _) = raw_payload.deconstruct();
+
+		Ok(UncheckedExtrinsic::new_signed(
+			call.deconstruct(),
+			signer.into_account().into(),
+			signature.into(),
+			extra,
+		))
+	}
+}
+
+impl ChainWithMessages for AssetHubWestend {
+	const TO_CHAIN_MESSAGE_DETAILS_METHOD: &'static str =
+		bp_asset_hub_westend::TO_ASSET_HUB_WESTEND_MESSAGE_DETAILS_METHOD;
+	const FROM_CHAIN_MESSAGE_DETAILS_METHOD: &'static str =
+		bp_asset_hub_westend::FROM_ASSET_HUB_WESTEND_MESSAGE_DETAILS_METHOD;
+}
+
+impl ChainWithRuntimeVersion for AssetHubWestend {
+	const RUNTIME_VERSION: Option<SimpleRuntimeVersion> =
+		Some(SimpleRuntimeVersion { spec_version: 1_016_001, transaction_version: 6 });
```

Comment:
for AHR/W we don't use version guard, but let's be aligned https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs#L137C16-L140

```suggestion
		Some(SimpleRuntimeVersion { spec_version: 1_017_007, transaction_version: 16 });
```